becca-new-app/src/App.js

```diff
@@ -1,18 +1,20 @@
-import React from 'react';
-
+import * as React from 'react';
+todoList[
+  { id: 1, title: 'Read over assignments'},
+  { id: 2, title: 'Exercise for 1 hour'},
+  { id: 3, title:  'Make dinner'}
+];
 function App() {
-  return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+return (
+<div>
+<h1>Todo List</h1>
+<ul>
+  {list.map(function(item){
+    return <li> {item.title}</li>
```

Comment:
Unnecessary space between `<li>` and `{item.title}` 

If you are using an IDE like vscode I would def recommend a formatter like prettier as an extension

Reply:
You're also missing something here. What do we need to add to the list element?